syntax = "proto3";
option cc_enable_arenas = true;

package NFq.NConfig;
option java_package = "ru.yandex.kikimr.proto";

////////////////////////////////////////////////////////////

message TAccessServiceConfig {
    bool Enable = 1;
    string Endpoint = 2;
    string PathToRootCA = 3;
}

message TControlPlaneProxyConfig {
    bool Enabled = 1;
    string RequestTimeout = 2;
    bool EnablePermissions = 3;
    string MetricsTtl = 4;
    string ConfigRetryPeriod = 31;
    TAccessServiceConfig AccessService = 32;
}
